examples/kubernetes/Makefile

```diff
@@ -0,0 +1,38 @@
+include ../../Makefile.defs
+
+ifeq ($(CILIUM_VERSION),)
+    CILIUM_VERSION = "$(shell cat ../../VERSION)"
+endif
+
+all: transform cilium.yaml
+
+%.sed:
+	for k8s_version in 1.7 1.8 1.9 1.10 1.11; do \
+	    (mkdir -p $$k8s_version && \
+	    cd $$k8s_version && \
+	    sed -f transforms2sed.sed ../templates/v1/$@ | \
+	    sed s+__CILIUM_VERSION__+$(CILIUM_VERSION)+g > "$*"); \
+	done
+
+%.yaml:
+	for k8s_version in 1.7 1.8 1.9 1.10 1.11; do \
+	    (mkdir -p $$k8s_version && \
+	    cd $$k8s_version && \
+	    cp ../templates/v1/$@ $@); \
+	done
+
+cilium.yaml:
+	for k8s_version in 1.7 1.8 1.9 1.10 1.11; do \
+        (cd $$k8s_version && \
+            rm -f ./$@ && \
+            for f in *.yaml; do (cat "$${f}"; echo "---") >> $@; done); \
+	done
+
+clean:
+	for k8s_version in 1.7 1.8 1.9 1.10 1.11; do \
+        rm ./$$k8s_version/*.yaml; \
```

Comment:
do we need a `cleanall` / `veryclean` to rm the directories as well created in .yaml and .sed?

Reply:
You can ignore if not really required.